packages/ods/src/components/toggle/src/components/ods-toggle/ods-toggle.scss

```diff
@@ -0,0 +1,94 @@
+@use '../../../../../style/focus';
+
+.ods-toggle {
+  &__container {
+    display: inline-block;
+    position: relative;
+    width: 48px;
+    height: 24px;
+
+    &__slider {
+      $slider: &;
+
+      position: absolute;
+      inset: 0;
+      border-radius: 1rem;
+      background-color: var(--ods-color-neutral-200);
+      cursor: pointer;
+
+      &::before {
+        position: absolute;
+        bottom: 2px;
+        left: 2px;
+        transition: all .4s ease;
+        z-index: 1;
+        border-radius: 1rem;
+        box-shadow: 0 1px 1px 0#000;
+        background-color: var(--ods-color-neutral-000);
+        width: 20px;
+        height: 20px;
+        content: '';
+      }
+
+      &:not(#{$slider}--disabled):hover {
+        background-color: var(--ods-color-neutral-300);
+
+        & #{$slider}__label {
+          color: var(--ods-color-neutral-800);
+        }
+      }
+
+      &--disabled {
+        background-color: var(--ods-color-neutral-100);
+        cursor: not-allowed;
+
+        & #{$slider}__label {
+          color: var(--ods-color-neutral-600);
+        }
+      }
+
+      &__label {
+        position: absolute;
+        top: 6px;
```

Comment:
Label is not more centered on FF (mac), you should not rely on px valiue to center a label as it may change between browser/os

Try something like that:
```
// This enforce the drawing zone take the whole height of the container and to start from the right sidr
position: absolute;
top: 0;
right: 0;
bottom: 0;

// Flex is the easiest way to ensure consistent vertical centering, and it works here because we works with a full height element (thx to previous block)
display: flex;
align-items: center;

// Then add a consistent spacing between the edge of the input and you label
padding-right: $toggle-label-spacing; // 4px seems to be ok
```

For ON:
```
// replace right: 0 with left: 0;
// replace padding-right: $toggle-label-spacing; with padding-left: $toggle-label-spacing;
```